checks/class-file-check.php

```diff
@@ -68,6 +68,19 @@ public function check( $php_files, $css_files, $other_files ) {
 
 		$musthave = array( 'index.php', 'style.css', 'readme.txt' );
 
+		$fse_find = array_filter( array_keys( $other_files ), function( $file_name ) {
+			if ( false !== stripos( $file_name, 'templates/index.html' ) || false !== stripos( $file_name, 'block-templates/index.html' ) ) {
+				return true;
```

Comment:
Will this not return true for the exception that was brought up on Trac here: https://core.trac.wordpress.org/ticket/54910,
classic themes that has a templates folder with a "silence is golden" index.html file.

Reply:
It probably will :-) but I haven't seen any themes adding it in that way

We could probably add in the mix a check for the `full-site-editing` tag, but it's not future proof...

Reply:
[wp_is_block_theme](https://github.com/WordPress/wordpress-develop/blob/0a6c37ceb0a86bebf0b287ac12d200e67bbf948f/src/wp-includes/class-wp-theme.php#L1486) has similar implementation.